clone gophercon workshops (#3414)
pulumi/pulumi-hugo: themes/default/content/resources/friend-or-foe-ai-devops-developers/index.md

---
# Name of the webinar.

> title: GopherCon

Reviewer: ```suggestion
title: AI, Friend or Foe | AI Talks for DevOps
```

> title: AI, Friend or Foe | AI Talks for DevOps

Reviewer: ```suggestion
title: "AI: Friend or Foe | AI Talks for DevOps"
```

> meta_desc: We are heading to GopherCon 2023. Meet us in-person at the Marriott Marquis in San Diego, or request a virtual 1:1 with one of our experts.

Reviewer: ```suggestion
meta_desc: Join us on November 8 in San Francisco for panel discussions about the impact of AI in DevOps and software development.
```

> meta_desc: Join us on November 8 in San Francisco for panel discussions about the impact of AI in DevOps and software development.

Reviewer: ```suggestion
meta_desc: Join us to explore how AI impacts DevOps and how we build, deploy, and manage infrastructure and software applications. [in person | San Francisco]
```

> meta_desc: Join us to explore how AI impacts DevOps and how we build, deploy, and manage infrastructure and software applications. [in person | San Francisco]

Reviewer 2: ```suggestion
meta_desc: Join us to explore how AI impacts DevOps and how we build, deploy, and manage infrastructure and software applications. (In-person, San Francisco)
```

# A featured webinar will display first in the list.
featured: false

# If the video is pre-recorded or live.
pre_recorded: false

# If the video is part of the PulumiTV series. Setting this value to true will list the video in the "PulumiTV" section.
pulumi_tv: false

# The preview image will be shown on the list page.
preview_image: ""

# Webinars with unlisted as true will not be shown on the webinar list
unlisted: false

# Gated webinars will have a registration form and the user will need
# to fill out the form before viewing.
gated: false

# The layout of the landing page.
type: webinars

# External webinars will link to an external page instead of a webinar
# landing/registration page. If the webinar is external you will need
# set the 'block_external_search_index' flag to true so Google does not index
# the webinar page created.
external: true
block_external_search_index: true

# The url slug for the webinar landing page. If this is an external
# webinar, use the external URL as the value here.

> url_slug: "https://info.pulumi.com/gophercon-2023"

Reviewer: ```suggestion
url_slug: "https://info.pulumi.com/ai-talks-for-devops-software-developers"
```

> url_slug: "https://info.pulumi.com/ai-talks-for-devops-software-developers"

Reviewer: ```suggestion
url_slug: "https://info.pulumi.com/sf-ai-talks-for-devops-software-developers"
```

# The content of the hero section.
hero:
    # The title text in the hero. This also serves as the pages H1.
    title: ""
    # The image the appears on the right hand side of the hero.
    image: "/icons/containers.svg"

# Webinar pages support multiple session via the 'multiple' property.
# multiple:
#   - datetime: 2020-02-05T10:00:00-07:00
#     hubspot_form_id: ""
#     gotowebinar_key: ""

# Content for the left hand side section of the page.
main:
    # Webinar title.
    title: ""
    # URL for embedding a URL for ungated webinars.
    youtube_url: ""
    # Sortable date. The datetime Hugo will use to sort the webinars in date order.

>     sortable_date: 2023-09-26T09:00:00-07:00

Reviewer: ```suggestion
    sortable_date: 2023-11-08T17:00:00-07:00
```

    # Duration of the webinar.

>     duration: "3 days"

Reviewer: ```suggestion
    duration: "1 day"
```
